src/test/java/com/wikia/webdriver/common/core/Assertion.java

```diff
@@ -153,4 +155,17 @@ public static void assertStringNotEmpty(String current) {
         assertion
     );
   }
+
+  /**
+   * This method checks that URL equals current URL and logs result The method is case-insensitive
+   */
+  public static void assertUrlEqualToCurrentUrl(WebDriver driver, String url) {
+    String currentUrl = driver.getCurrentUrl().toLowerCase();
+    PageObjectLogging.log(
+        "Log Url",
+        "Url " + url + " is equal to current Url " + currentUrl,
+        "Url " + url + " isn't equal to current Url " + currentUrl,
```

Comment:
I was thinking about it recently — you're logging here (`Url foo isn't equal to current Url bar` → `false`), right? Isn't it confusing, should then the URLs be different, or should they be equal? (`Url foo is equal to current Url bar` → `false`) would be a truthful statement which is internally cohesive and tells user what was expected and why it failed.

We should decide on one variant and use it widely and consistently.

What do you think?